tasks/18.txt

```diff
@@ -1,3 +1,10 @@
 Create a SQL query to calcualte total orders count by each day in 1998:
 
-| Order Date | Total Number of Orders |
\ No newline at end of file
+| Order Date | Total Number of Orders |
+
+//task18
+db.orders.aggregate([
+     { $match: {OrderDate: { $regex: /^1998/ }}},
+     { $sort: {OrderDate:-1}},
+     { $group: { _id: "$OrderDate" , TotalNumberOfOrders : { $sum: 1 } } }
+ ])
```

Comment:
Please run the query before
db.orders.find().forEach(order=>{
    order.OrderDate = new Date(order.OrderDate)
    order.RequiredDate = new Date(order.RequiredDate)
    order.ShippedDate = new Date(order.ShippedDate)
    db.orders.save(order)
    })